### PR TITLE
Adding RUG::B meeting

### PR DESCRIPTION
**Pull request by halfbyte**
As RUG::B will happen in the wwwtf week by chance, I thought it might be worthwhile to add it to this list.

This has been cleared with @pragtob, who's the current organizer, on twitter.

### CODE REVIEW

**Comment by PragTob** [comment]
Not sure what the possible values are here, but you don't really need to rsvp - we don't check that. It's more an optional nice to know thing :D 

thanks @halfbyte !

**Comment by halfbyte** [comment]
@PragTob Made the event "free". Didn't really think about it first, tbqh.

My pleasure :)

File: _posts/2017-05-04-rug-b-meetup.mdown [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+layout: post
+title:  "Ruby Usergroup Berlin May Meetup"
```
Comment (hagenburger) [inline _posts/2017-05-04-rug-b-meetup.mdown]: The CI department comments: it should be written “Ruby User Group Berlin” ;)

**Review by hagenburger — CHANGES_REQUESTED** [review]
Thanks, @halfbyte :)

**Comment by rmehner** [comment]
Welcome my Ruby friends! :)

**Comment by afknapping** [comment]
img_5449

Sorry, cannot inspect rn, screenshot from iphone 6s @ 10.1.1

**Comment by halfbyte** [comment]
Yes, the text also is too long for the desktop view. I didn't write that text, though.

No one was using that type of ticket-type before, I guess. I could venture a fix (replace the text with only "Free" would work, I guess), but this is more of a general issue with the wwwtf site.

Quick fix without changing something at the template level would be to switch back to RSVP. 

Any Idea what would be best, @rmehner (paging you as you have merged this)

**Comment by rmehner** [comment]
We all don't have so much time on our hands, so "replace the text with only "Free" would work" is probably good :)

**Comment by halfbyte** [comment]
@rmehner See #75. 

Thanks all!

**Pull request merged**